enterprise/api/v1/views/enterprise_group.py

```diff
@@ -109,19 +116,27 @@ def get_learners(self, *args, **kwargs):
                 'previous': null,
                 'results': [
                     {
-                        'learner_uuid': 'enterprise_customer_user_id',
-                        'pending_learner_id': 'pending_enterprise_customer_user_id',
-                        'enterprise_group_membership_uuid': 'enterprise_group_membership_uuid',
+                        'learner_id': integer or None,
+                        'pending_learner_id': integer or None,
+                        'enterprise_group_membership_uuid': UUID,
+                        'enterprise_customer': EnterpriseCustomerSerializer,
```

Comment:
this would be an object of enterprise customer details

Reply:
removing this from serializer